.github/workflows/android.yml

```diff
@@ -143,7 +143,6 @@ jobs:
                 -B "${CMAKE_BUILD_DIR}" \
                 -G Ninja \
                 -D CMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }} \
-                -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
Is that a deliberate change or a leftover from a try?

Reply:
Deliberate change aimed at reducing delta of differences between local docker builds and CI builds

Reply:
Do we want to keep this then

https://github.com/opengisch/QField/blob/93a92cc9adb81af98e1a7a7745f2ccd6dfda36e2/.github/workflows/android.yml#L118-L122